test/new-e2e/tests/npm/cilium_lb_conntracker_test.go

```diff
@@ -0,0 +1,219 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package npm
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/DataDog/agent-payload/v5/process"
+	"github.com/pulumi/pulumi-kubernetes/sdk/v4/go/kubernetes"
+	corev1 "github.com/pulumi/pulumi-kubernetes/sdk/v4/go/kubernetes/core/v1"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments"
+	awskubernetes "github.com/DataDog/datadog-agent/test/new-e2e/pkg/provisioners/aws/kubernetes"
+	"github.com/DataDog/test-infra-definitions/common/config"
+	npmtools "github.com/DataDog/test-infra-definitions/components/datadog/apps/npm-tools"
+	"github.com/DataDog/test-infra-definitions/components/datadog/kubernetesagentparams"
+	kubeComp "github.com/DataDog/test-infra-definitions/components/kubernetes"
+	"github.com/DataDog/test-infra-definitions/components/kubernetes/cilium"
+	"github.com/DataDog/test-infra-definitions/components/kubernetes/istio"
+)
+
+type ciliumLBConntrackerTestSuite struct {
+	e2e.BaseSuite[environments.Kubernetes]
+
+	httpBinService *corev1.Service
+}
+
+func TestCiliumLBConntracker(t *testing.T) {
+	// TODO: find a way to update this list dynamically
+	versionsToTest := []string{"1.15.14", "1.16.7", "1.17.1"}
+	for _, v := range versionsToTest {
+		v := v
```

Comment:
This should be useless since Go 1.22: https://go.dev/doc/go1.22#language

Reply:
Cool, thanks